.github/workflows/ci.yml

```diff
@@ -0,0 +1,407 @@
+      - scan-vulnerability-php
+      - scan-vulnerability-prometheus-exporter-file
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download Images
+        uses: actions/download-artifact@v2
+        with:
+          name: docker-image-prometheus-exporter-file
+          path: ./tmp
+      - run: docker load --input ./tmp/image*.tar
+      - run: docker images
+      - run: make ci-docker-login push-prometheus-exporter-file
+        env:
+          DOCKER_USER: ${{ secrets.HUB_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.HUB_PASSCODE }}
+  push-http:
+    name: Push nginx ${{ matrix.nginx }}
+    if: (github.event_name == 'push' || github.event_name == 'schedule') && github.ref == 'refs/heads/master'
+    needs:
+      - test-http
+      - test-php
+      - test-prometheus-exporter-file
+      - scan-vulnerability-http
+      - scan-vulnerability-php
+      - scan-vulnerability-prometheus-exporter-file
+      - generate-nginx-strategy
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        nginx: ${{ fromJson(needs.generate-nginx-strategy.outputs.nginx) }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download Images
+        uses: actions/download-artifact@v2
+        with:
+          name: docker-image-http-${{ matrix.nginx }}
+          path: ./tmp
+      - run: docker load --input ./tmp/image*.tar
+      - run: docker images
+      - run: make ci-docker-login push-http
+        env:
+          DOCKER_USER: ${{ secrets.HUB_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.HUB_PASSCODE }}
+  push-php:
+    name: Push PHP ${{ matrix.php }} for ${{ matrix.type }} on Alpine ${{ matrix.alpine }}
+    if: (github.event_name == 'push' || github.event_name == 'schedule') && github.ref == 'refs/heads/master'
+    needs:
+      - test-http
+      - test-php
+      - test-prometheus-exporter-file
+      - scan-vulnerability-http
+      - scan-vulnerability-php
+      - scan-vulnerability-prometheus-exporter-file
+      - generate-alpine-strategy
+      - generate-php-strategy
+      - generate-php-type-strategy
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        alpine: ${{ fromJson(needs.generate-alpine-strategy.outputs.alpine) }}
+        php: ${{ fromJson(needs.generate-php-strategy.outputs.php) }}
+        type: ${{ fromJson(needs.generate-php-type-strategy.outputs.type) }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Download Images
+        uses: actions/download-artifact@v2
+        with:
+          name: docker-image-${{ matrix.type }}-${{ matrix.php }}-${{ matrix.alpine }}
+          path: ./tmp
+      - run: docker load --input ./tmp/image*.tar
+      - run: docker images
+      - run: make ci-docker-login push-${{ matrix.type }}
+        env:
+          DOCKER_USER: ${{ secrets.HUB_USERNAME }}
+          DOCKER_PASSWORD: ${{ secrets.HUB_PASSCODE }}
```

Comment:
Before we can merge this PR we need these two secrets to be able to push freshly created images to Docker Hub

Reply:
Added `DOCKERHUB_USER` and `DOCKERHUB_TOKEN` to `secrets`. Please rename accordingly.